docs/sources/setup-grafana/set-up-https.md

```diff
@@ -0,0 +1,254 @@
+---
+aliases:
+  - 
+keywords:
+  - grafana
+  - https
+  - ssl
+  - certificates
+title: Set up Grafana HTTPS for Secure Web Traffic
+weight: 900
+---
+
+# Set up Grafana HTTPS for Secure Web Traffic
+
+The Grafana UI is accessed through the web. Setting up HTTPS ensures that
+communication going back and forth between Grafana and the end user is encrypted, to include
+login information and metric data.
+
+In order to accomplish this, Grafana needs a key to use for encrypting the traffic, and a
+[Secure Socket Layer (SSL) certificate](https://www.kaspersky.com/resource-center/definitions/what-is-a-ssl-certificate) that helps establish the identity of the site. With these two things, 
+users can see the proper browser "lock icon" indicating their connection is secure.
```

Comment:
```suggestion
In order to ensure secure traffic over the internet, Grafana must have a key for encryption as well as a [Secure Socket Layer (SSL) Certificate](https://www.kaspersky.com/resource-center/definitions/what-is-a-ssl-certificate) to verify the identity of the site. This will display a browser "lock icon" which confirms the connection is safe.
```